components/vocabularies/hpo/annotations/src/test/java/org/phenotips/vocabulary/internal/hpoannotations/NegativePhenotypeForDiseaseAnnotationsExtensionTest.java

```diff
@@ -83,6 +105,17 @@
     @Mock
     private Vocabulary vocabulary;
 
+    @Mock
+    private XWiki xwiki;
+
+    @Mock
+    private XWikiDocument document;
+
+    @Mock
+    private BaseObject object;
```

Comment:
I would rename this to relocationObject, since it is only used as such. It is harder to understand the code when there are generic names like `object` around